--- node-v16.18.1/configure~	2022-11-04 11:22:04.000000000 +0000
+++ node-v16.18.1/configure	2022-11-08 00:05:45.388064121 +0000
@@ -1,9 +1,10 @@
-#!/bin/sh
+#!/bin/sh
 
 # Locate an acceptable Python interpreter and then re-execute the script.
 # Note that the mix of single and double quotes is intentional,
 # as is the fact that the ] goes on a new line.
 _=[ 'exec' '/bin/sh' '-c' '''
+command -v python3.11 >/dev/null && exec python3.11 "$0" "$@"
 command -v python3.10 >/dev/null && exec python3.10 "$0" "$@"
 command -v python3.9 >/dev/null && exec python3.9 "$0" "$@"
 command -v python3.8 >/dev/null && exec python3.8 "$0" "$@"
--- node-v16.18.1/configure~	2022-11-08 00:05:45.000000000 +0000
+++ node-v16.18.1/configure	2022-11-08 00:09:38.087436882 +0000
@@ -23,7 +23,7 @@
   from distutils.spawn import find_executable as which
 
 print('Node.js configure: Found Python {}.{}.{}...'.format(*sys.version_info))
-acceptable_pythons = ((3, 10), (3, 9), (3, 8), (3, 7), (3, 6))
+acceptable_pythons = ((3, 11), (3, 10), (3, 9), (3, 8), (3, 7), (3, 6))
 if sys.version_info[:2] in acceptable_pythons:
   import configure
 else:
