diff -ru node-v12.18.3~/src/node.cc node-v12.18.3/src/node.cc
--- node-v12.18.3~/src/node.cc	2020-07-22 15:26:54.000000000 +0000
+++ node-v12.18.3/src/node.cc	2020-07-30 00:19:21.327740286 +0000
@@ -902,7 +902,7 @@
       per_process::cli_options->use_largepages == "silent") {
     int result = node::MapStaticCodeToLargePages();
     if (per_process::cli_options->use_largepages == "on" && result != 0) {
-      fprintf(stderr, "%s\n", node::LargePagesError(result));
+      // fprintf(stderr, "%s\n", node::LargePagesError(result));
     }
   }
 
